xla/service/gpu/build_defs.bzl

```diff
@@ -1,8 +1,10 @@
 """ GPU-specific build macros.
 """
 
-load("@local_config_cuda//cuda:build_defs.bzl", "cuda_library")
-load("@local_config_rocm//rocm:build_defs.bzl", "rocm_copts")  # copybara:comment
+load("@local_config_cuda//cuda:build_defs.bzl", 
+            "if_cuda_is_configured", "cuda_library")
+load("@local_config_rocm//rocm:build_defs.bzl", 
```

Comment:
if you can try putting the `local_config_rocm` statement on one line, I think that should fix the copybara problem

Reply:
@ddunl  I put everything into one line and also removed '# copybara:comment' at the end, but there is still a copybara error. Could you check what's wrong ? I am not able to open the link..